document-generation/utils/sampleTestData.ts

```diff
@@ -0,0 +1,410 @@
+        },
+        {
+          periodType: "OPTION",
+          periodUnitCount: 3,
+          periodUnit: "MONTH",
+          optionOrder: 1,
+        },
+        {
+          periodType: "OPTION",
+          periodUnitCount: 30,
+          periodUnit: "DAY",
+          optionOrder: 4,
+        },
+      ],
+      popStartRequest: true,
+      requestedPopStartDate: "2022-09-30",
+      timeFrame: "NO_SOONER_THAN",
+      recurringRequirement: null,
+    },
+    gfeOverview: {
+      gfeOrGfpFurnished: true,
+      propertyAccountable: true,
+      dpasUnitId: "888999",
+      dpasCustodianNumber: "909090",
+      propertyCustodianName: "Jane Smith",
+    },
+    contractConsiderations: {
+      packagingShippingNoneApply: true,
+      packagingShippingOther: true,
+      packagingShippingOtherExplanation: "N/A",
+      potentialConflictOfInterest: false,
+      conflictOfInterestExplanation: "None so far.",
+      contractorProvidedTransfer: true,
+      contractorRequiredTraining: true,
+      requiredTrainingServices: ["basic training", "intermediate training", "advanced training"],
+    },
+    section508AccessibilityStandards: {
+      piiPresent: true,
+      workToBePerformed: "Investigation of specific individuals",
+      systemOfRecordName: "Secret Name",
+      FOIACityApoFpo: "Crystal",
+      FOIACountry: "United States of America",
+      FOIAStreetAddress1: "973 Inspector Rd",
+      FOIAStreetAddress2: "",
+      FOIAAddressType: "US",
+      FOIAZipPostalCode: "22222",
+      FOIAStateProvinceCode: "VA",
+      FOIAFullName: "Alice Wonder",
+      FOIAEmail: "alice@ccpo.mil",
+      BAARequired: false,
+      potentialToBeHarmful: false,
+      section508Sufficient: null,
+      accessibilityReqs508: "Some requirements for 508.",
+    },
+  },
+};
+
+export const sampleIgceRequest = {
+  documentType: "INDEPENDENT_GOVERNMENT_COST_ESTIMATE",
+  templatePayload: {
+    surgeCapabilities: 34,
+    fundingDocument: {
+      fundingType: "FS_FORM",
+      orderNumber: "O2206-097-097-184790",
+      gtcNumber: "A2201-097-097-18092",
+    },
+    periodsEstimate: [
+      {
+        period: { periodUnit: "YEAR", periodUnitCount: "1", periodType: "BASE", optionOrder: "" },
+        periodLineItems: [
+          {
+            clin: "3000",
+            idiqClin: "3000 Cloud Support UNCLASSIFIED",
+            dowTaskNumber: "4.2.2.1",
+            serviceOffering: "Cloud Audit/Monitoring Tools",
+            itemDescriptionOrConfigSummary: "Monitoring network data",
+            monthlyPrice: 394.38,
+            monthsInPeriod: 36,
+          },
+          {
+            clin: "2000",
+            idiqClin: "2000 Cloud SECRET",
+            dowTaskNumber: "4.2.2.2",
+            serviceOffering: "Special custom built app - not SNOW",
+            itemDescriptionOrConfigSummary: "To handle special cases that can not be done in SNOW",
+            monthlyPrice: 799.0,
+            monthsInPeriod: 36,
+          },
```

Comment:
This doesn't match the schema, which uses underscores. If we are switching to camelCase we need to update `hoth_api.yaml`.

Reply:
> I was doing some testing with the example schema provided in the ticket, did we change the schema recently?
The example is in camelCase while the schema still reads with underscores?

I changed the implementation schema in this ticket, but not in the `hoth_api.yaml` since I thought it was okay. Initially, I think most of the implementation was done with `camelCase` even though the spec was in `snake_cake` (slightly loose from the spec).

Very good point though, but it does make sense that they should match up.

Reply:
Will address in a separate PR.